lib/shoryuken/queue.rb

```diff
@@ -14,8 +14,8 @@ def initialize(client, name)
 
     def visibility_timeout
       client.get_queue_attributes(
-        queue_url: url,
-        attribute_names: ['VisibilityTimeout']
+          queue_url:       url,
+          attribute_names: ['VisibilityTimeout']
       ).attributes['VisibilityTimeout'].to_i
```

Comment:
There's no specific code style, it's more like just to keep the style it currently uses.